packages/bpk-component-popover/src/BpkPopover.tsx

```diff
@@ -300,7 +303,23 @@ const BpkPopover = ({
                       {label}
                     </span>
                   )}
-                  <div className={bodyClassNames}>{children}</div>
+                  <div className={bodyClassNames}>
+                    <div>{children}</div>
+                    {!labelAsTitle && closeButtonIcon && (
+                      <BpkCloseButton
```

Comment:
In the code above we also define a closebutton, can we ensure both close buttons us the same code (ie. refactor to a const)

Reply:
yeah, it's the same code, let me do some refactoring. Thank you